README.md

```diff
@@ -1,6 +1,12 @@
-HTTP Server Handler
-===================
+HTTP Server Request Handlers
+==============
 
-Provides the `RequestHandlerInterface` of [PSR-15][psr-15].
+This repository holds the `RequestHandlerInterface` related to [PSR-15 (HTTP Server Request Handlers)][psr-url].
 
-[psr-15]: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-15-request-handlers.md
+Note that this is not a Server Request Handler implementation of its own. It is merely the interface that describe a Server Request Handler.
+
+You can find [implementations][implementation-url] and [installation instructions][package-url] for the specification on the packagist.
```

Comment:
Same changes as https://github.com/php-fig/http-factory/pull/9 should be applied here.